app/assets/stylesheets/regions/mastheads/slider-masthead.scss

```diff
@@ -1 +1,70 @@
-/* New slider masthead styles will go here */
\ No newline at end of file
+/*
+ * Slider v201611
+ */
+div.slider.version-201611 {
+  margin: 0 auto;
+  width: 1130px;
+  height: 440px;
+
+  div.grid {
+    .column {
+      float: left;
+      position: relative;
+      height: 440px;
+      overflow: hidden;
+      font-family: $font-stack-heading;
+
+      &.image {
+        width: 780px;
+
+        div.photo-by {
+          position: absolute;
+          bottom: 0;
+          left: 0;
+          padding: 16px;
+          font-size: 16px;
+          color: $white;
+          background-color: rgba(56,56,56,0.90);
+        }
+      }
+
+      &.header {
+        width: 350px;
+        color: $white;
+
+        div.aligned {
+          position: absolute;
```

Comment:
@mirabelrice I agree blindly nesting like the article describes is bad (see line 5 of the Zen: `Flat is better than nested.`), but that's not what's really happening here. 

Notice, there's no empty rules. Rather, I'm targeting a specific case, e.g. the new slider master, to the exclusion of existing slightly variant versions. I've inherited a deeply nested structure here with the flex-slider. The SASS rules target specific pieces of it.

But I'm always open to cleaner alternative, provided they don't hide complexity or produce conflicts.